src/app/hearing/components/view-hearing/view-hearing.component.ts

```diff
@@ -16,17 +13,22 @@ import { NoteType } from '../../../notes/models/note-type';
 import { NotesService } from '../../../notes/services/notes.service';
 import { formatDuration } from '../../../utils/date-utils';
 import { Status } from '../../../core/reference/models/status.model';
+import { PossibleHearingActionsService } from '../../services/possible-hearing-actions.service';
 
 @Component({
   selector: 'app-view-hearing',
   templateUrl: './view-hearing.component.html',
-  styleUrls: ['./view-hearing.component.scss']
+  styleUrls: ['./view-hearing.component.scss'],
+  providers: [PossibleHearingActionsService]
 })
 export class ViewHearingComponent implements OnInit {
-  hearingId: string
+  hearingId: string;
   hearing: Hearing;
-  hearingActions = HearingActions
+  hearingActions = HearingActions;
   @ViewChild(MatSelect) actionSelect: MatSelect;
+  objectKeys = Object.keys;
```

Comment:
plz compute *possibleAction* once and do not keep reference to Object.keys function
```js
this.this.possibleActions = objectKeys(possibleActions)
```
and then just iterate over it in html file